gprofiler/profilers/java.py

```diff
@@ -69,6 +77,18 @@ def get_ap_version() -> str:
     return Path(resource_path("java/async-profiler-version")).read_text()
 
 
+def _locate_hotspot_error_file(process: Process) -> str:
+    error_file = "hs_err_pid%p.log"
+    for arg in process.cmdline():
+        if arg.startswith("-XX:ErrorFile"):
+            _, error_file = arg.split("=", maxsplit=1)
+
+    error_file = error_file.replace("%p", str(process.pid))
+    if not error_file.startswith("/"):
+        error_file = f"{process.cwd()}/{error_file}"
+    return error_file
```

Comment:
Also check `/tmp`?

```

In addition, if the -XX:ErrorFile=file flag is not specified, the system attempts to create the file in the working directory of the process. In the event that the file cannot be created in the working directory (insufficient space, permission problem, or other issue), the file is created in the temporary directory for the operating system
```

(from that link https://docs.oracle.com/javase/7/docs/webnotes/tsg/TSG-VM/html/felog.html)

Reply:
added